src/pages/patientView/PatientViewPage.tsx

```diff
@@ -247,6 +275,40 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
             });
         }
 
+        if (this.patientIdsInCohort && this.patientIdsInCohort.length > 0) {
+            const indexInCohort = this.patientIdsInCohort.indexOf(this.patientId);
+            cohortNav = (<TablePaginationControls
+                            showItemsPerPageSelector={false}
+                            showFirstPage={true}
+                            showLastPage={true}
+                            textBetweenButtons={`${indexInCohort+1} of ${this.patientIdsInCohort.length} patients`}
+                            firstPageDisabled={indexInCohort === 0}
+                            previousPageDisabled={indexInCohort === 0}
+                            nextPageDisabled={indexInCohort === this.patientIdsInCohort.length-1}
+                            lastPageDisabled={indexInCohort === this.patientIdsInCohort.length-1}
+                            onFirstPageClick={() =>
+                                window.location.href = this.getURL(
```

Comment:
next week lets try to handle this without page refresh.  that's the holy grail and it shouldn't be too hard.